web/templates/change_email_modal.hbs

```diff
@@ -1,4 +1,5 @@
 <form id="change_email_form">
+    <p>{{t "You will receive a confirmation email at the new address you enter."}}</p>
```

Comment:
This seems to have several fundamentally independent changes. Please put each on its own commit; this is really important for being able to review your work.

Check out our [GitHub guide](https://zulip.readthedocs.io/en/latest/git-guide.html) and [commit guidelines](https://zulip.readthedocs.io/en/latest/contributing/commit-discipline.html) for more details.

Reply:
I initially thought about making a PR for this, but I opted to combine all the commits into one and absolutely forgot to check the commit guidelines.